decidim-core/lib/tasks/decidim_reminders_tasks.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+namespace :decidim do
+  namespace :reminders do
```

Comment:
Please add it in https://github.com/decidim/decidim/blob/develop/docs/modules/install/pages/index.adoc#scheduled-tasks